test/dynamic_color_plugin_test.dart

```diff
@@ -8,15 +10,35 @@ void main() {
   test('getCorePalette', () async {
     final sampleFromListCorePalette = generateCorePalette((i) => i);
 
-    DynamicColorTestingUtils.setMockDynamicColors(sampleFromListCorePalette);
+    DynamicColorTestingUtils.setMockDynamicColors(
+      colors: sampleFromListCorePalette,
+    );
     final colors = await DynamicColorPlugin.getCorePalette();
     expect(colors, sampleFromListCorePalette);
   });
 
   test('getCorePalette returns null', () async {
-    DynamicColorTestingUtils.setMockDynamicColors(null);
-
+    DynamicColorTestingUtils.setMockDynamicColors(
+      colors: null,
+    );
     final colors = await DynamicColorPlugin.getCorePalette();
     expect(colors, equals(null));
   });
+
+  test('getControlAccentColor', () async {
+    const color = Color.fromARGB(12, 24, 123, 53);
```

Comment:
Just a random value so we can know none of the channels have tampered.

Reply:
Nice